src/Core/Components/List/FluentAutocomplete.razor

```diff
@@ -14,11 +14,11 @@
                          AutoComplete="@AutoComplete"
                          Appearance="@Appearance"
                          Disabled="@Disabled"
-                         Placeholder="@(this.SelectedOptions?.Any() == false ? Placeholder : string.Empty)"
+                         Placeholder="@(SelectedOptions?.Any() is not true ? Placeholder : string.Empty)"
```

Comment:
After checking Unit Tests, the result is not the same using `== false` and `is not true` when SelectedOptions is null (I think). In fact, the actual Unit Tests don't contain a `Placeholder` not defined, and with this change, the Unit Tests contains a string empty `Placeholder`.

It's probably better to render a `null` value using a code like this (to verify)
`Placeholder="@(SelectedOptions?.Any() is not true ? Placeholder : null)"`

Reply:
I'll double check that logic in the unit test. When there were null VS .count == 0 selected items, it would show/hide the placeholder. I think the valid logic is null OR 0 items should show the placeholder. I'll update the test

Reply:
Reverted the breaking change - looks like the placeholder shows fine now - probably related to the selected options changes